### ISSUE TITLE
Copyright statement not visible after upgrade

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#6830]
**Describe the bug**
After upgrading to OJS 3.2, the copyright statement is not visible anymore even when it was already assigned. The dashboard shows that everything is correct, but it doesn't work.

**To Reproduce**
1. Upgrade to OJS 3.2
2. Enable the copyright for the journal
2. Click on any article 
3. See the error: no copyright statement

**What application are you using?**
 OJS 3.2

**Additional information**
I've figured out that it is a case-sensitive issue. In the `publication_settings` table, the `setting_name` field is storing values with a case that Themes doesn't recognize. It should be `licenseUrl` and currently is `licenseURL`.\
It is possible to fix it by running the following query:

```
UPDATE publication_settings
SET 
    `setting_name` = REPLACE(`setting_name`,
        'URL',
        'Url');
```
However, maybe a DB sanitization to ensure this value is correct could prevent future issues.

**Comment by asmecher** [comment]
@israelcefrin, what version was the upgrade from? What release of OJS 3.2 was the upgrade to (which build or github branch)?

**Comment by israelcefrin** [comment]
The more recent case I was upgrading to a 3.2.1.3, see below the full previous version history:
![image](https://user-images.githubusercontent.com/4992902/110169708-13dac380-7dc7-11eb-8d76-e80443334d6f.png)

**Comment by asmecher** [comment]
@israelcefrin, the upgrade from 3.1.x to 3.2.x happened at the 3.2.1-1 release, before the fix at https://github.com/pkp/ojs/commit/3d3df3932abe29479be84f58cbe436b52ea07f51 was applied. The fix is present in 3.2.1-3, but that upgrade step is not executed because the 3.1.x to 3.2.x migration has already happened. You can resolve it afterwards by manually running the SQL query in the commit.
